### PR TITLE
docs / ETH requirement + API key permissions

### PR DESCRIPTION
**Pull request by dennisocana**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**A description of the changes proposed in the pull request**:

- removed 0.05 ETH balance requirement for participating in liquidity bounty program
- added information to recommend "trade only" enabled API keys and suggest not using "transfer" or "withdraw" enabled API keys
- modified screenshot in Coinbase Pro connector guide to avoid confusion

### CODE REVIEW

File: documentation/docs/connectors/binance.md [hunk 1]
```diff
@@ -30,11 +30,15 @@ Because Binance is a centralized exchange, you will need to generate and provide
 <br />
 5 - Now you have created an API key. Please note that to trade on Binance using Hummingbot, **Enable Trading** must be selected.
 
+!!! warning "API key permissions"
+    For general use of Hummingbot, we recommend "trade only" enabled API keys and suggest not using "transfer" or "withdraw" enabled API keys.
```
Comment (chowryan) [inline documentation/docs/connectors/binance.md]: We recommend using only "trade" enabled API keys; enabling "withdraw" is unnecessary for current Hummingbot strategies.

File: documentation/docs/installation/api-keys.md [hunk 2]
```diff
@@ -11,3 +11,6 @@ Please see below for instructions to find your API keys for the exchanges that H
 * [Huobi Global](/connectors/huobi/#creating-huobi-api-keys)
 
 * [IDEX](/connectors/idex/#api-key)
+
+!!! warning "API key permissions"
+    For general use of Hummingbot, we recommend "trade only" enabled API keys and suggest not using "transfer" or "withdraw" enabled API keys.
```
Comment (chowryan) [inline documentation/docs/installation/api-keys.md]: same as above

File: documentation/docs/connectors/coinbase.md [hunk 3]
```diff
@@ -21,7 +21,10 @@ Because Coinbase Pro is a centralized exchange, you will need to generate and pr
 
 ![coinbase2](/assets/img/coinbase2.png)
 
-Make sure you give permissions to **View** and **Trade** (**Transfer** is optional), and enter your 2FA code.
+Make sure you give permissions to **View** and **Trade**, and enter your 2FA code.
+
+!!! warning "API key permissions"
+    For general use of Hummingbot, we recommend "trade only" enabled API keys and suggest not using "transfer" or "withdraw" enabled API keys.
```
Comment (chowryan) [inline documentation/docs/connectors/coinbase.md]: same as above

File: documentation/docs/connectors/binance.md (hunk 1 above)
Comment (dennisocana) [reply documentation/docs/connectors/binance.md]: Rephrased. Thanks!

**Pull request merged**